beckend/database-models/UserModel.ts

```diff
@@ -0,0 +1,43 @@
+const user = (sequelize, DataTypes) => {
+    const User = sequelize.define('user', {
+        user_id: DataTypes.STRING,
+        login: DataTypes.STRING,
+        password: DataTypes.STRING,
+        age: DataTypes.INTEGER,
+        isDeleted: DataTypes.BOOLEAN
+    });
+
+    User.findById = async id => {
```

Comment:
async modificator is overhead, because User.findOne returns Promise.
`User.findById= user => User.create(user);`